### ISSUE TITLE
global inland lake

### ISSUE THREAD

**Issue by bbakernoaa** [toddkarin/global-land-mask#3]
is there a mask out there for global inland lakes? this would be a useful feature as well.

**Comment by toddkarin** [comment]
Yes, that would be useful. Unfortunately, I don't have time to work on this now.
